ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -458,7 +458,7 @@ RnsShell::LayerInvalidateMask  RSkComponentTextInput::updateComponentProps(const
   oskLaunchConfig_.theme=RSkToSdkOSKeyboardTheme(textInputProps.traits.keyboardAppearance);
   oskLaunchConfig_.returnKeyLabel=RSkToSdkOSKReturnKeyType(textInputProps.traits.returnKeyType);
   oskLaunchConfig_.enablesReturnKeyAutomatically=textInputProps.traits.enablesReturnKeyAutomatically;
-  oskLaunchConfig_.placeHolderName=placeholderString_;
+  oskLaunchConfig_.placeHolderName=textInputProps.placeholder.c_str();
```

Comment:
is c_str() operation required ? because textInputProps.placeholder and placeHolderName both are of same type std::string

Reply:
Rightly pointed. Will correct it.